lightning/src/ln/channelmanager.rs

```diff
@@ -6982,7 +6915,6 @@ impl<'a, Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref>
 			}
 		}
 
-		let last_node_announcement_serial: u32 = Readable::read(reader)?;
```

Comment:
Are there any backwards compatibility issues here?

Reply:
I think I'm missing something here... Wouldn't an older `ChannelManager` have written this field, so we need to read and ignore it still?